experimental/dds/attributable-map/src/test/memory/.mocharc.cjs

```diff
@@ -12,7 +12,7 @@ module.exports = {
 	"fgrep": ["@Benchmark", "@MemoryUsage"],
 	"node-option": ["expose-gc", "gc-global", "unhandled-rejections=strict"], // without leading "--"
 	"recursive": true,
-	"reporter": "@fluid-tools/benchmark/dist/MochaMemoryTestReporter.js",
+	"reporter": "@fluid-tools/benchmark/dist/MochaReporter.js",
```

Comment:
I think at this point we can revert all the changes outside the tool; these ones and the lockfile ones.

Reply:
But I have deleted `MochaMemoryTestReporter` since `MemoryBenchmarkStats` is gone